src/components/_icons.scss

```diff
@@ -6,94 +6,62 @@
 //
 // See https://github.com/nhsevidence/nice-icons
 .fa {
-  @include icon-base;
-
-  &.icon-android:before {
-    content: nice-icon(android);
+  &.icon-android {
+    @include nice-icon(android);
   }
 
-  &.icon-apple:before {
-    content: nice-icon(apple);
+  &.icon-apple {
+    @include nice-icon(apple);
```

Comment:
This bumps up the size of the output CSS quite a lot, byt changing from content to mixin. Probably not a big deal after gzipping but wonder if using the content approach is more performant?

Reply:
The base mixin is now `nice-icons-base` if you need it